tasks/main.yml

```diff
@@ -254,18 +252,27 @@
     mode: "750"
   when: item
   with_items:
-    - "{{ paperlessng_consumption_dir }}"
     - "{{ paperlessng_data_dir }}"
     - "{{ paperlessng_trash_dir }}"
     - "{{ paperlessng_media_root }}"
     - "{{ paperlessng_staticdir }}"
+    - "{{ paperlessng_export_dir }}"
+
+- name: create paperless-ngx consumption directory and set permissions
+  file:
+    path: "{{ paperlessng_consumption_dir }}"
+    state: directory
+    owner: "{{ paperlessng_system_user }}"
+    group: "{{ paperlessng_system_group }}"
+    mode: "750"
+  when: consumption_mount_exist == false
 
 - name: rename initial config
   command:
     cmd: "mv -f {{ paperlessng_directory }}/paperless.conf {{ paperlessng_directory }}/paperless.conf.template"
     removes: "{{ paperlessng_directory }}/paperless.conf"
 
-- name: configure paperless-ng
+- name: configure paperless-ngx
   lineinfile:
     path: "{{ paperlessng_directory }}/paperless.conf.template"
```

Comment:
This file doesn't exist.  It appears to be skipping any installation if nothing exists already.

Reply:
Same as above. The flags were wrong on the register current state step. I was testing the upgrade to 1.8.0. Both flags now set to follow the fresh installation path